data/curated/new_submission/palmtrees.md

```diff
@@ -0,0 +1,31 @@
+|variable                |class         |description                           |
+|:-----------------------|:-------------|:-------------------------------------|
+|spec_name               |character     |Taxonomic name of species (binomial nomenclature) following the World Checklist of palms. |
+|acc_genus               |character     |Accepted genus name from the World Checklist of palms. |
+|acc_species             |character     |Accepted species name from the World Checklist of palms. |
+|palm_tribe              |character     |Name of palm tribe from the World Checklist of palms. |
+|palm_subfamily          |character     |Name of palm subfamily from the World Checklist of palms. |
+|climbing                |factor<bfb5a> |Whether palm species has climbing habit or not, or both if populations vary in this trait. |
+|acaulescent             |factor<7af82> |Whether palm species has an acaulescent growth form (leaves and inflorescence rise from the ground, i.e. lacking a visible aboveground stem) or not, or both if populations vary in this trait. |
+|erect                   |factor<36b70> |Whether palm species has an erect stem (rather than an acaulescent or climbing growth form) or not, or both if local populations vary in this trait. |
```

Comment:
```suggestion
|erect                   |factor |Whether palm species has an erect stem (rather than an acaulescent or climbing growth form) or not, or both if local populations vary in this trait. |
```